lms/views/dashboard/api/user.py

```diff
@@ -91,14 +91,22 @@ def __init__(self, request) -> None:
         schema=ListUsersSchema,
     )
     def students(self) -> APIStudents:
-        _, students_query = self._students_query(
+        roster_lat_updated, students_query = self._students_query(
             assignment_ids=self.request.parsed_params.get("assignment_ids"),
             segment_authority_provided_ids=self.request.parsed_params.get(
                 "segment_authority_provided_ids"
             ),
         )
+        # students_query is a query object, we need to paginate it
```

Comment:
The right fix here, the one that will fix this longer term is to settle on LMSUser for all queries.

Because `report` still uses the old tables it's not an easy change. But we can start to migrate the code and "hide" the older tables in the services.